pkg/sql/sem/tree/schedule.go

```diff
@@ -16,15 +16,20 @@ type FullBackupClause struct {
 	Recurrence Expr
 }
 
+type ScheduleLabelSpec struct {
```

Comment:
All exported structs and methods require a comment that starts with the exported name:

`ScheduleLabelSpec` describes the labelling specification for a scheduled job.

Reply:
done